tokio/src/sync/semaphore.rs

```diff
@@ -96,21 +88,21 @@ impl Semaphore {
     }
 
     /// Acquires permit from the semaphore.
```

Comment:
It might be nice if this comment said something about errors, like
```suggestion
    /// Acquires a permit from the semaphore.
    /// 
    /// If the semaphore has been closed, this returns an [`AcquireError`].
    /// Otherwise, this returns a [`SemaphorePermit`] representing the 
    /// acquired permit.
```
or something?